src/Provider/PackageArchiver.php

```diff
@@ -145,12 +153,12 @@ public function archive_updates( $value ) {
 			$slug = $update_data[ $type ];
 
 			// Bail if the package isn't whitelisted.
-			if ( ! $manager->has_package( $slug, $type ) ) {
+			if ( ! $this->satispress->contains( [ 'slug' => $slug, 'type' => $type ] ) ) {
```

Comment:
In this case, I think `$this->satispress` would be easy to comprehend if it was called `$this->whitelisted_packages` (assuming my understanding of the new architecture is correct. 

`satispress` as the repository name means having to do a mental jump as to what it actually represents.

Reply:
I'm definitely not set on that naming, but was trying to move away from the whitelist terminology a bit. In the future, I could see a different UI where plugins and themes are added to the repository instead of just ticking a few checkboxes like now. It could also be possible to define virtual packages by filling out a form. Maybe something like `custom_repository` works better?

Reply:
I think `custom_repository` is even worse :-)

How about `exposed_packages`?

Reply:
Haha, I never claimed to be good at naming things. Whitelist seems to make the most sense in this context, so I'll stick with that.